### DISCUSSION TITLE
Troubles with registration to Copilot

### DISCUSSION THREAD

**Discussion by ZhenyaGoroh1** [community/community#70688] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi all. I tried to register with Copilot. I entered data such as First Name, Last Name, Address and so on, then an error was immediately displayed to me.

> Cannot proceed with payment
There appears to be an issue with the billing information. We're looking into it and will send you an update within 48 hours. Thank you for your patience.

I didn't enter my card details. What could this mean?

**Comment by joshmatni** [comment]
Hey ZhenyaGoroh,

I would say, based on the error message you encountered, it seems there might be a billing issue on Copilot's end, despite you not entering your card details yet. It's advisable to double-check the information you entered for any mistakes or incomplete fields, as small errors can sometimes trigger system alerts. The error message mentioned an update within 48 hours, so it might be worth waiting for that. If the issue persists or you don’t receive an update, reaching out to Copilot's customer support could provide further assistance. Additionally, you might want to check Copilot's FAQ section or community forums for similar experiences shared by other users. It's possible that this is a temporary system issue, so attempting registration again later could also be an option. Lastly, don’t forget to check your email spam folder for any communications from Copilot regarding this issue. Hope this helps!

**Reply by ZhenyaGoroh1** [reply]
I can't reenter or check my information for now. I can only see the error message.

**Reply by joshmatni** [reply]
Ah, that sounds frustrating. It's weird that it won't let you check or change your info. For now, it might just be a waiting game until Copilot gets back to you. In the meantime, maybe shoot their support a message about what's going on. They might have a quick fix. And who knows, someone on their forums might’ve run into the same issue and have some tips. Keep an eye on your email too, in case they reach out with updates. Hope it gets sorted out soon!

**Reply by ZhenyaGoroh1** [reply]
Is copilot blocked in some countries?

**Reply by joshmatni** [reply]
According to this link: https://github.com/orgs/community/discussions/47749 , Basically, a bunch of countries got banned from using Copilot. Including Cuba, Iran, North Korea, Syria, Russia, and Belarus.

**Reply by ZhenyaGoroh1** [reply]
Oh, it's a pity( I'm from Belarus.

**Reply by joshmatni** [reply]
Ah, that's a bummer! Alternatively, you could reach out to GitHub's support to get a clearer understanding of the restrictions and if there are any workarounds. It's always worth a shot! I hope I was able to answer your question!

**Reply by ZhenyaGoroh1** [reply]
Thank you very much for your time and advice!

**Comment by PushoDev** [comment]
En Cuba tampoco podemos utilizarlo, a pesar de ser una herramienta necesaria
